### PR TITLE
docs: DIDComm Router/Mediator documentation

### LINKED ISSUE #1115: DIDComm Router/Mediator documentation
DIDComm Router/Mediator documentation - Part of #1109.

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #1115 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: docs/didcomm_router.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Aries DIDComm Router/Mediator
+
+The aries-framework-go project can be used a DIDComm Router/Mediator. The agents that do not
```
Comment (troyronda) [inline docs/didcomm_router.md]: As a

File: docs/didcomm_router.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Aries DIDComm Router/Mediator
+
+The aries-framework-go project can be used a DIDComm Router/Mediator. The agents that do not 
+have inbound transport can register with the router. Basically, the agent asks another agent to 
+route the messages to it by asking the permission. On successful grant, agent receives the
```
Comment (troyronda) [inline docs/didcomm_router.md]: for permission.

File: docs/didcomm_router.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Aries DIDComm Router/Mediator
+
+The aries-framework-go project can be used a DIDComm Router/Mediator. The agents that do not 
+have inbound transport can register with the router. Basically, the agent asks another agent to 
+route the messages to it by asking the permission. On successful grant, agent receives the 
+endpoint and routing key details. These details are used in DID Exchange Invitation or DID 
+Document Service Descriptor.
+
+## Edge Agents without Inbound Capability
+The project supports DIDComm between two agents without inbound capability through router. The
```
Comment (troyronda) [inline docs/didcomm_router.md]: a router

**Review by troyronda — APPROVED** [review]

File: docs/didcomm_router.md (hunk 1 above)
Comment (rolsonquadras) [reply docs/didcomm_router.md]: [Updated](https://github.com/hyperledger/aries-framework-go/compare/523808246463b15b8a0e880d8cbc9cb889558772..3fe47f63bb2fb52c165715cd5072a3a12a272d8f).

File: docs/didcomm_router.md (hunk 2 above)
Comment (rolsonquadras) [reply docs/didcomm_router.md]: [Updated](https://github.com/hyperledger/aries-framework-go/compare/523808246463b15b8a0e880d8cbc9cb889558772..3fe47f63bb2fb52c165715cd5072a3a12a272d8f).

File: docs/didcomm_router.md (hunk 3 above)
Comment (rolsonquadras) [reply docs/didcomm_router.md]: [Updated](https://github.com/hyperledger/aries-framework-go/compare/523808246463b15b8a0e880d8cbc9cb889558772..3fe47f63bb2fb52c165715cd5072a3a12a272d8f).

**Pull request merged**